Firebase/Core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.3.3 -- M59
+- [changed] Carthage and zip file distributions are now built with Xcode 11.
```

Comment:
Add a sentence that Xcode 10.x is no longer supported with this release from the zip file and Carthage.

Reply:
Done, updated internal CL as well.